tsconfig.json

```diff
@@ -1,5 +1,6 @@
 {
   "compilerOptions": {
+    "baseUrl": ".",
```

Comment:
Avoids a warning when running Cypress.

Reply:
Something in the line of "no baseUrl defined, so TypeScript paths won't be supported" ... but we don't have `"paths"` in `tsconfig`, so it's meaningless.